alpm-utils/src/lib.rs

```diff
@@ -6,15 +6,20 @@
 
 #[cfg(feature = "conf")]
 mod conf;
+
 #[cfg(feature = "alpm")]
+mod alpm;
 mod db;
+
 /// Utils for dependency checking.
 #[cfg(feature = "alpm")]
 pub mod depends;
 mod target;
 
 #[cfg(feature = "conf")]
 pub use crate::conf::*;
+
 #[cfg(feature = "alpm")]
+pub use crate::alpm::*;
```

Comment:
The `alpm` module was previously left out of the module tree, and so was never compiled. I've added it back and fixed the compilation errors.

Reply:
This shouldn't be a pub mod but a mod and pub use.